mocha-multi.js

```diff
@@ -293,6 +293,14 @@ function mochaMulti(runner, options) {
 
 class MochaMulti {
   constructor(runner, options) {
+    debugger;
```

Comment:
I guess this is left over from development?

Reply:
🤦